app/templates/ember-community-survey-2020.hbs

```diff
@@ -5,114 +5,375 @@
   </p>
-
-  <div class="mt-3">
-    <a
-      class="es-button"
-      href="https://tilde.wufoo.com/forms/2020-emberjs-community-survey/"
-      rel="noopener noreferrer"
-      target="_blank"
-    >
-      Take the survey now
-    </a>
-  </div>
 </EmberCommunitySurvey::Introduction>
 
 <EmberCommunitySurvey::Section
   @hasDarkBackground={{false}}
-  @sectionId="about-the-survey"
-  @sectionTitle="About the Survey"
+  @sectionId="recommending-ember"
+  @sectionTitle="Recommending Ember"
 >
+  <:charts>
+    <Highcharts
+      @chartOptions={{this.SS_Q101.options}}
+      @data={{this.SS_Q101.data}}
+    />
+  </:charts>
+
   <:body>
-    <div class="layout">
-      <div class="lg:col-2">
-        <img
-          alt="Tomster and Friends"
-          class="survey-image"
-          src="/images/survey/photo1.jpg"
-        >
-
-        <h3>
-          Totally Anonymous
-        </h3>
-
-        <p>
-          No personal identification is attached to survey responses. Last year, over 1300 participants shared information about themselves, their businesses, and applications.
-        </p>
-      </div>
+    <p>
+      We asked respondents how likely they were to recommend Ember to others, on a scale of 1-5. The mean recommendation score is 4; approximately 80% of people who use Ember are highly likely to recommend it to others.
+    </p>
+    <p>
+      Participants reported their reasoning for their score as free-form text, and below, we've provided a sample of responses alongside their corresponding recommendation score.
+    </p>
+    <p>
+      As in previous years, we did not observe a great deal of variety in the written responses that corresponded to values of three or less. Similarly, we saw the opposite effect in the upper ranges—that is, <strong>community members simply have a lot of different reasons to recommend Ember</strong>!
+    </p>
+    <p>
+      Here is a flavor of what reasons participants gave for their scoring:
+    </p>
 
-      <div class="lg:col-2">
-        <img
-          alt="Emberconf Crowd"
-          class="survey-image"
-          src="/images/survey/photo2.jpg"
-        >
+    <ul class="list-unstyled scoring-list">
+      <EsCard class="my-2">
+        <p class="text-center large">
+          Score: 5
+        </p>
 
-        <h3>
-          Open until March 8th
-        </h3>
+        <blockquote class="quote">
+          Ember has been a solid choice for our team. The conventions allow us to manage several products with a handful of devs. It contains everything needed to successfully deliver out of the box, or within a handful of highly refined community offerings, like <a href="">concurrency</a> or <a href="">mirage</a>.
```

Comment:
Is it okay to have a link inside a blockquote? If we're not going to redirect a user to external sites, let's remove the anchor tags?

```suggestion
          Ember has been a solid choice for our team. The conventions allow us to manage several products with a handful of devs. It contains everything needed to successfully deliver out of the box, or within a handful of highly refined community offerings, like ember-concurrency or Mirage.
```

Reply:
Ahh I meant to link to the addons so that's my bad.